fine_tuning_sql/notebooks/1_download_data.ipynb

```diff
@@ -5,26 +5,73 @@
    "metadata": {},
    "source": [
     "# Part 1: Download data\n",
-    "This notebook goes over data download for SQL model fine-tuning"
+    "This notebook goes over data download for SQL model [Pretraining](#pretraining-dataset) and [Fine-tuning](#fine-tunning-dataset)\n",
+    "\n",
+    "We recommend you going directly to the [Fine-tuning](#fine-tunning-dataset) if you want to use as base model an instruct model like Llama-7b-Instruct-hf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pretraining dataset\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Pretraining dataset: Load and save \n"
+    "### Small stack smol dataset for SQL Pretraining\n",
+    "- [bigcode/the-stack-smol](https://huggingface.co/datasets/bigcode/the-stack-smol)\n",
+    "- Total samples: 155813472\n",
+    "- Total dataset size 902MB\n",
+    "- SQL dataset Size 158MB\n",
+    "- The jsonl files contains ```prompt``` and ```completions``` pairs per sample. For fine-tuning, both ```prompt``` and ```completions``` fields are populated."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Full squad dataset for SQL\n",
+    "For downloading the dataset you will need to request access in [huggingface bigcode/the-stack-smol](https://huggingface.co/datasets/bigcode/the-stack-smol)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from datasets import load_dataset\n",
```

Comment:
import os missing here. I think this will fail

Reply:
fixed